File: rss-reader/app/src/main/java/com/abubusoft/kripton/examples/rssreader/api/VogellaApi.java
package com.abubusoft.kripton.examples.rssreader.api;

import com.abubusoft.kripton.examples.rssreader.model.RSSFeed;

import retrofit2.Call;
import retrofit2.http.GET;

public interface VogellaApi {

    @GET("article.rss")
    Call<RSSFeed> loadRSSFeed();
}

File: kotlin-rss-reader/app/src/main/java/com/abubusoft/kripton/example/rssreader/service/api/VogellaApi.kt
package com.abubusoft.kripton.example.rssreader.service.api


import com.abubusoft.kripton.example.rssreader.service.model.RssFeed
import retrofit2.Call
import retrofit2.http.GET

interface VogellaApi {

    @GET("article.rss")
    fun loadRSSFeed(): Call<RssFeed>
}